package.json

```
@@ -1,7 +1,7 @@
 {
   "name": "realm-studio",
   "productName": "MongoDB Realm Studio",
-  "version": "10.0.0-mongodb.0",
+  "version": "10.0.0",
```

Shouldn't this be a beta version like all other SDKs?
```suggestion
  "version": "10.0.0-beta.1",
```

---

The more I think about it, I think you're right.
And I also think this should just release on the `beta` channel, instead of a separate one for `mongodb`. I originally choose a different channel because we already had a beta channel with the core 6 changes, but now that this has released on latest (the success of that is still debatable) the `beta` channel can be repurposed.